.env.example

```diff
@@ -9,4 +9,8 @@ NEXT_PUBLIC_WEB3_STORAGE_DID=""
 WEB3_STORAGE_PROOF=""
 NEXT_PUBLIC_AIRSTACK_API_KEY=""
 NEXT_PUBLIC_GITCOIN_PASSPORT_API_KEY=""
-NEXT_PUBLIC_GITCOIN_SCORER_ID=""
\ No newline at end of file
+NEXT_PUBLIC_GITCOIN_SCORER_ID=""
+NEXT_PUBLIC_SUPABASE_URL=""
+NEXT_PUBLIC_SUPABASE_ANON_KEY=""
+SUPABASE_SERVICE_ROLE_KEY=""
+SUPABASE_JWT=""
```

Comment:
@Seroxdesign , 
Just want to confirm that these two keys i.e., `SUPABASE_SERVICE_ROLE_KEY` and  `SUPABASE_JWT`  are also added in Vercel. 

Right?